### PR TITLE
Add tests for CloseClaimCommandParser

### PR DESCRIPTION
**Pull request by matthewyip1511**
Adding tests for CloseClaimCommandParser to increase our test coverage. 

While testing it was also discovered that the CloseClaimCommand was somehow missing an equals method, so that has been added in as well 😅

### CODE REVIEW

File: src/main/java/seedu/address/logic/commands/CloseClaimCommand.java [hunk 1]
```diff
@@ -86,4 +86,21 @@ public CommandResult execute(Model model) throws CommandException {
...
     }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof CloseClaimCommand)) {
```
Comment (yanqiyqh) [inline src/main/java/seedu/address/logic/commands/CloseClaimCommand.java:97]: lets keep it consistent with AddClaimCommand's equals method
`if (other instanceof CloseClaimCommand) { return ... }
return false`

**Review by yanqiyqh — COMMENTED** [review]
overall lgtm, just one nitpick 👍

File: src/main/java/seedu/address/logic/commands/CloseClaimCommand.java (hunk 1 above)
Comment (matthewyip1511) [reply src/main/java/seedu/address/logic/commands/CloseClaimCommand.java:97]: Good spot, thanks 🙂 will make the changes

**Review by yanqiyqh — APPROVED** [review]
LGTM!

**Pull request merged**